packages/shared/src/api/chatApi.ts

```diff
@@ -78,11 +78,17 @@ export const markChatRead = (whom: string) =>
 export const sendDirectMessage = async (
```

Comment:
Should combine this with `postsApi.sendPost`, following the same basic pattern as `postsApi.getChannelPosts`